src/Data/Fin/Induction.agda

```diff
@@ -59,6 +59,20 @@ open WF public using (Acc; acc)
   induct {suc i} (acc rec) = Pᵢ⇒Pᵢ₊₁ i (induct (rec (inject₁ i) i<i+1))
     where i<i+1 = ℕ<⇒inject₁< (i<1+i i)
 
+≤-Induction : {k : Fin (suc n)} (P : ∀ {p} → k ≤ p → Set ℓ)
+  (pk : P ≤-refl)
+  (ps : ∀ {m k≤m} → P {inject₁ m} k≤m → P (≤-inj→suc k≤m))
+  → ∀ {m} k≤m → P {m} k≤m
```

Comment:
Could you make sure the code conforms to the [indentation style guide](https://github.com/agda/agda-stdlib/blob/master/notes/style-guide.md) please? Oh and the section on variable names!

Reply:
I indented and put better variable names.